src/UI/Implementation/Component/Button/Factory.php

```diff
@@ -55,4 +55,11 @@ public function month($default) {
 	public function bulky($icon_or_glyph, $label, $action) {
 		return new Bulky($icon_or_glyph, $label, $action);
 	}
+
+	/**
+	 * @inheritdoc
+	 */
+	public function toggle($label, $action, $action_deactivated) {
+		return new Toggle($label, $action, $action_deactivated);
+	}
```

Comment:
I think we are missing some means to spawn the button in a activated (or deactivated) state here. Also I cannot see which one of these is the default.

Reply:
Sorry, I do not understand your first sentence. Could you explain it a little bit more exact to me?
Regarding the default state: Is your idea to pass it as parameter through the constructor?

Reply:
If I use this method and get a toggle button back: How do I know if it is activated or deactivated? If I need it to be deactivated: How can I accomplish that?